step4/index.html

```diff
@@ -70,26 +70,33 @@
 }
 </style>
 <canvas id="canvas" width="512" height="512"></canvas>
-<a href="#" id="link">Download</a>
```

Comment:
step4 introduces the "Download" link, so maybe re-add it (with the `<br />`)

Yup, managing several files' worth of changes is a pain :(

Reply:
It's actually introduced in step 5, hence the removal here

Reply:
OR everything's offset by the intro step and I need to rename the folders.

Reply:
Yes, I think I considered the intro step to be zero. But of course in the Codelabs interface it's not. So uh... sorry.